server/src/Metrics.hs

```diff
@@ -51,9 +54,17 @@ createAndRegisterIcepeakMetrics = IcepeakMetrics
                               "Total number of bytes written to the journal so far."))
   <*> register (gauge
     (Info "icepeak_subscriber_count" "Number of websocket subscriber connections."))
+  <*> register (gauge (Info "icepeak_internal_queue_length" "Length of the internal queue."))
+  <*> register (histogram (Info "icepeak_sync_duration" "Duration of a Sync command.")
+                          syncBuckets)
+  <*> register (histogram (Info "icepeak_http_req_handling_duration"
+                                "Duration of the handling of an HTTP request.")
+                          httpBuckets)
   where
     requestCounter = counter (Info "icepeak_http_requests"
                                    "Total number of HTTP requests since starting Icepeak.")
+    syncBuckets = exponentialBuckets 0.0000008 1.2 20
+    httpBuckets = exponentialBuckets 0.00001 1.5 20
```

Comment:
These seem to small if I understand the docs of `exponentialBuckets` correctly. I think anything smaller than 1ms is probably not interesting for us. In other services, for HTTP request handler durations we seem to use [`defaultBuckets`](https://hackage.haskell.org/package/prometheus-client-1.1.0/docs/src/Prometheus.Metric.Histogram.html#defaultBuckets).

For syncing, we might need our own scale though, the range we care about is probably between 1ms and 1s.